task/attest/src/main.rs

```diff
@@ -70,16 +75,57 @@ fn load_data_from_region<
     }
 }
 
+// the size of the measurements we record
+// NOTE: the rust crypto digest traits don't expose consts for the lengths
+// of various hash functions
+const SHA3_256_DIGEST_SIZE: usize =
+    <Sha3_256Core as OutputSizeUser>::OutputSize::USIZE;
+
+// the number of Measurements we can record
+const CAPACITY: usize = 16;
```

Comment:
Is this carefully calculated or "yeah that should be enough"

Reply:
the later for the most part, still in the 'educated guess' phase